src/tree_sortable.rs

```diff
@@ -49,6 +51,15 @@ impl FromGlib<i32> for SortColumn {
     }
 }
 
+impl fmt::Display for SortColumn {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "SortColumn::{}", match *self {
+            SortColumn::Default => "Default",
+            SortColumn::Index(_) => "Index",
```

Comment:
Don't print column number?

Reply:
`Debug` does it. :)

Reply:
This is not a `Display` impl anyway but you know my opinion about that ;)

In this case it should print things like "Default column", "Column at index {}", etc. and not basically the same as the `Debug` impl.

Reply:
Yes I know. But the current way of implementing `Display` is this one. However I agree this is not very nice... I'm open to suggestions for next release!

Reply:
My suggestion is that you can't automatically implement `Display` as it requires context specific knowledge about the type that we don't have.

For this specific type a correct `Display` implementation would be the one I mentioned.